### PR TITLE
Issue 758 root resource

### PR DESCRIPTION
**Pull request by attrobit**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.0.x`. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee
Fixes #) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR
Fix for [issue 758](https://github.com/OpenAPITools/openapi-generator/issues/758)

The fix is backward compatible.

### Technical committee
@bbdouglas @JFCote @sreeshas @jfiala @lukoyanov @cbornet @jeff9finger 

I would be grateful for a quick feedback.

### CODE REVIEW

File: modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSSpecServerCodegenTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ */
+public class JavaJAXRSSpecServerCodegenTest {
+    
+    private JavaJAXRSSpecServerCodegen instance;
+    
+    @Before
+    public void before() {
+        instance = new JavaJAXRSSpecServerCodegen();
+    }
+
```
Comment (JFCote) [inline modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSSpecServerCodegenTest.java]: You could remove a line here

Comment (attrobit) [reply modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSSpecServerCodegenTest.java]: done.

Comment (attrobit) [reply modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSSpecServerCodegenTest.java]: @JFCote Whats next? I cant merge the PR. Do you merge the PR?

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by attrobit** [comment]
@wing328 Thanks for your comment. I changed the name and email address for my commits. Is it good now?

**Comment by wing328** [comment]
@attrobit the commits are now correctly linked. The fix looks good to me.

@JFCote thanks for reviewing the change.

**Pull request merged**